items/public-announcements

```diff
@@ -0,0 +1,18 @@
+{
+	"name": "Public Announcements",
+	"version": "1.0",
+	"author_username": "aniika",
+	"release_date": "13.04.2019",
+	"license": "MIT",
+	"price_usd": "0",
+	"download_url": "https://github.com/naplaya/public-announcements/archive/master.zip",
```

Comment:
Please create a release in Github with the version and update the links for download.

Reply:
sry dont understand what you mean with update link. this link download the file  if visited in browser

Reply:
The user is going to download the master branch from the repository with that link, so if you do push to the master branch is NOT going to be consistent with the version 1.0. You can create a release like I do in Bludit https://github.com/bludit/bludit/releases and you are going to have a link only for the version 1.0. You get my point ?